dietpi/func/dietpi-banner

```diff
@@ -31,36 +31,36 @@
 
 	aDESCRIPTION=(
 
-		'Device model'
-		'Uptime'
-		'CPU temp'
-		'FQDN/hostname'
-		'NIS domainname'
-		'LAN IP'
-		'WAN IP'
-		'Freespace (RootFS)'
+		'Device model        '
+		'Uptime              '
+		'CPU temp            '
+		'FQDN/hostname       '
+		'NIS domainname      '
+		'LAN IP              '
+		'WAN IP              '
+		'Freespace (RootFS)  '
 		'Freespace (userdata)'
-		'Weather (wttr.in)'
-		'Custom banner entry'
+		'Weather (wttr.in)   '
+		'Custom banner entry '
```

Comment:
@Fourdee 
Note that this entry is customizable. Perhaps we can full up the returned value from the input box (below) with white spaces to match 20 characters?

Reply:
@MichaIng 

Hmmm, personally I believe revert back to no alignment for now due to :
https://github.com/MichaIng/DietPi/pull/2649#issuecomment-475673381

?

Reply:
@Fourdee 
Yeah I think it's okay for now. Space calculation time (e.g. via `$(( 20 - ${#var} ))` ) for this two strings (Custom banner entry as well) should be only marginal compared to the external commands we use to obtain banner values, especially those including www access (weather/WAN IP).

But lets move this fine tuning + performance testing to v6.23